docs/pages/authentication/id-token.mdx

```diff
@@ -0,0 +1,378 @@
+---
+meta:
+  title: "Authenticate with ID tokens"
+  parentTitle: "Authentication"
+  description: "Learn more about ID token permissions"
+---
+
+ID token authentication allows Liveblocks to handle permissions for you. This
+means that when you create or modify a room, you can set a user’s permissions on
+the room itself, this acting as a source of truth. Later, when a user tries to
```

Comment:
"Groups" could be mentioned here as well, not necessarily explained but at least they'd know what to expect when clicking on the ID token page.

Reply:
Yes! Done it.